test/e2e_test.go

```diff
@@ -139,12 +129,39 @@ func TestE2EPutGetLogicForEigenDAStore(t *testing.T) {
 
 	t.Log("Setting input data on proxy server...")
 	commit, err := daClient.SetInput(ts.ctx, testPreimage)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	// 2 - fetch data from EigenDA for generated commitment key
 	t.Log("Getting input data from proxy server...")
 	preimage, err := daClient.GetInput(ts.ctx, commit)
-	assert.NoError(t, err)
-	assert.Equal(t, testPreimage, preimage)
+	require.NoError(t, err)
+	require.Equal(t, testPreimage, preimage)
+}
+
+func TestE2EPutGetLogicForMemoryStore(t *testing.T) {
+	if runTestnetIntegrationTests {
```

Comment:
Shouldn't this be the opposite?

Reply:
We want to run this test with the normal tests, and not run it with the testnet-integration tests.